client/src/components/AdminContainer/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Layout, Button, Menu } from 'antd';
+import logo from '../../assets/images/logo.png';
+import './style.css';
+
+const { Header } = Layout;
+
+const AdminContainer = ({ buttonContent, contentComponent }) => {
+  return (
+    <>
+      <Layout>
+        <Header>
+          <div className="header-image">
+            <img src={logo} alt="logo" />
+          </div>
+          <div className="header-btn">
+            {buttonContent !== undefined && (
+              <Button type="primary" danger>
+                {buttonContent}
+              </Button>
+            )}
+          </div>
+        </Header>
+        <div className="container">
+          <div className="container-side">
+            <div className="side-content">
+              <Menu>
+                <Menu.Item key="1">
+                  <Link exact="true" to="/admin">
+                    Home
+                  </Link>
+                </Menu.Item>
+                <Menu.Item key="2">
+                  <Link exact="true" to="/admin/cohorts">
+                    Cohorts
+                  </Link>
+                </Menu.Item>
+              </Menu>
+              <div className="side-content-btn">
+                <Button type="primary" danger>
+                  Logout
+                </Button>
+              </div>
+            </div>
+          </div>
+          <div className="container-content">{contentComponent}</div>
```

Comment:
@MohammedAlghazali React has special handling for `props.children` which you can use here, to make it easier to use AdminContainer.  You can replace `contentComponent` with `children` in this file, and then in other files you can nest JSX inside the AdminContainer tags and all of the things you nest will automatically be put into `props.children`.  So for example you could write this:
```
<AdminCohort buttonContent="Add Cohort">
  <Test location="hello from Admin Home Page" />
</AdminCohort>
```

Reply:
Also if you did that, you probably wouldn't need to have the `<Test>` component and could just use text or a `<h1>` directly.

Reply:
oh, great now I can delete Test component, it's nice
thank you